client/components/AddTestToQueueWithConfirmation/index.jsx

```diff
@@ -129,6 +133,37 @@ function AddTestToQueueWithConfirmation({
         );
     };
 
+    const renderErrorDialog = () => {
+        return (
+            <BasicModal
+                show={errorMessage}
+                title={'Conflicting Report Found'}
+                content={
+                    'The report could not be created because an existing report was found on the reports page with the same AT, browser and test plan version. Would you like to return the existing report back to the test queue?'
+                }
```

Comment:
The project doesn't enforce a maximum line length, but it appears to (mostly) split strings at the 100-column boundary. @howard-e @alflennik @stalgiag is there any consensus on this?

Reply:
It's certainly been mentioned a few times now,  in GitHub threads and in sync discussions. I've created an issue to now track that discussion, https://github.com/w3c/aria-at-app/issues/924.

Reply:
Hey we found out that the project formats code at column 80. I split the strings there.